src/components/CurriculumComponents/UnitsTab/UnitsTab.test.tsx

```diff
@@ -24,6 +24,13 @@ jest.mock("@/context/Analytics/useAnalytics", () => ({
 describe("components/pages/CurriculumInfo/tabs/UnitsTab", () => {
   beforeEach(() => {
     jest.clearAllMocks();
+    const mockIntersectionObserver = jest.fn();
```

Comment:
I'd move this out of the before each so it's not getting redeclared for each test